docs/concepts.rst

```diff
@@ -474,21 +474,21 @@ operations which can occur.
 
 :Addition:
 
-    An operation which creates a arrayset, sample, or some metadata which
+    An operation which creates a column, sample, or some metadata which
```

Comment:
metadata will not be part of the history?

Reply:
Will be addressed in #181